tests/tlsio_bearssl_ut/CMakeLists.txt

```diff
@@ -0,0 +1,29 @@
+#Copyright (c) Microsoft. All rights reserved.
+#Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+cmake_minimum_required(VERSION 2.8.11)
+
+set(theseTestsName tlsio_bearssl_ut)
+
+#set(MBED_ROOT_DIR $ENV{MBED_ROOT_DIR} CACHE PATH "")
+set(BEARSSL_ROOT_DIR "C:/Users/markrad/S/BearSSL")
```

Comment:
This should be a relative path ala $ENV{OpenSSLDir} CACHE PATH, etc.

Reply:
Yes indeed. I inadvertently left this residue in from my testing. I'll take a look at it.

---
In reply to: [305153694](https://github.com/Azure/azure-c-shared-utility/pull/292#discussion_r305153694) [](ancestors = 305153694)